bosh/cloud_config.yml

```diff
@@ -235,6 +256,11 @@ vm_extensions:
   cloud_properties:
     target_pool: drills-web-target-pool
 
+- name: wings-credhub-uaa-target-pool
+  cloud_properties:
+    target_pool: wings-credhub-uaa-target-pool
+
+
```

Comment:
Same, we can have a config like this:
```yaml
networks:
- name: internal-wings
  type: dynamic
  subnets:
  - az: z1
    cloud_properties:
      network_name: bosh
      subnetwork_name: internal-wings
      tags:
      - internal
      - wings-uaa-web
      - wings-credhub-web
vm_extensions:
- cloud_properties:
    target_pool: wings-credhub-uaa-target-pool
  name: wings-credhub-uaa-target-pool
vm_types:
- cloud_properties:
    cpu: 4
    ram: 8192
    root_disk_size_gb: 100
    root_disk_type: pd-ssd
    zone: us-central1-f
  name: uaa
```
and use `bosh update-config` to set it.